internal/api/drafts.go

```diff
@@ -540,6 +549,14 @@ func DraftsDocumentHandler(
 			l.Info("retrieved document draft", "doc_id", docId)
 
 		case "DELETE":
+			// Authorize request.
```

Comment:
Interesting, I thought this would already be in place for deleting documents. 😄

Reply:
We were! 🙂 We were only authorizing draft owners for all methods before this PR, and now with this PR we authorize owners+contributors for all methods, and this is extra authorization to only allow owners to delete drafts.